documentation/src/main/asciidoc/topics/con_endpoints_memcached.adoc

```diff
@@ -1,19 +1,27 @@
 = Memcached
-{brandname} provides an implementation of the Memcached text protocol for remote client access.
+{brandname} provides an implementation of the Memcached text protocol for
+remote client access.
 
-The {brandname} Memcached server supports clustering with replicated and
+[IMPORTANT]
+====
+The Memcached endpoint is deprecated and planned for removal in a future
```

Comment:
is memcached supported in the product? if not, I would make this page community only.

Reply:
@pruivo thanks I've made all the references to Memcached as community only